### PR TITLE
[WIP] Adds DISA STIG RHEL7 V1R1 ansible role

### PR DESCRIPTION
**Pull request by WesleyCeraso**
This commit adds the DISA STIG RHEL7 V1R1 ansible role.

The roles we output in "build/roles" directory contains the keyword ```tasks:```, but it raises an error when used as a role. I had to remove it.

Some tasks had fatal errors and stopped the whole process:
```
TASK [: Prevent user modification of GNOME idle-delay]
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "Destination /etc/dconf/db/local.d/locks/00-security-settings-lock does not exist !", "rc": 257}

TASK [: Prevent user modification of GNOME idle_activation_enabled] 
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "Destination /etc/dconf/db/local.d/locks/00-security-settings-lock does not exist !", "rc": 257}
	to retry, use: --limit @/home/wprudencio/workspace/ansible-role-stig-rhel7/tests/test.retry

TASK [: Prevent user modification of GNOME lock-enabled] 
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "Destination /etc/dconf/db/local.d/locks/00-security-settings-lock does not exist !", "rc": 257}
	to retry, use: --limit @/home/wprudencio/workspace/ansible-role-stig-rhel7/tests/test.retry

TASK [: Prevent user modification of GNOME lock-delay] 
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "Destination /etc/dconf/db/local.d/locks/00-security-settings-lock does not exist !", "rc": 257}
	to retry, use: --limit @/home/wprudencio/workspace/ansible-role-stig-rhel7/tests/test.retry

TASK [: Ensure SELinux State is Enforcing] 
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "policy is required if state is not 'disabled'"}
	to retry, use: --limit @/home/wprudencio/workspace/ansible-role-stig-rhel7/tests/test.retry

TASK [: Configure SELinux Policy] 
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "missing required arguments: state"}
	to retry, use: --limit @/home/wprudencio/workspace/ansible-role-stig-rhel7/tests/test.retry

TASK [: Prevent user modification of GNOME banner-message-enabled] 
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "Destination /etc/dconf/db/local.d/locks/00-security-settings-lock does not exist !", "rc": 257}
	to retry, use: --limit @/home/wprudencio/workspace/ansible-role-stig-rhel7/tests/test.retry

TASK [: Set rsyslog remote loghost] 
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "Destination /etc/rsyslog.conf does not exist !", "rc": 257}
	to retry, use: --limit @/home/wprudencio/workspace/ansible-role-stig-rhel7/tests/test.retry
```

I suppose we should not let any task have fatal errors, because the whole role becomes useless.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,14 @@
-Role Name
+STIG RHEL7 DISA
 =========
 
-A brief description of the role goes here.
+This profile contains configuration checks that align to the DISA STIG for Red Hat Enterprise Linux V1R1.
```
Comment (shawndwells) [inline README.md]: Assuming this was derived from SSG, it's V1R2. Just noticed the SSG profile needs to be updated as well... PR in flight for that.

File: README.md [hunk 2]
```diff
@@ -1,38 +1,14 @@
...
-    - hosts: servers
-      roles:
-         - { role: username.rolename, x: 42 }
-
-License
--------
-
-BSD
+This script is generated from an OpenSCAP profile without preliminary evaluation.
+It attempts to fix every selected rule, even if the system is already compliant.
```
Comment (shawndwells) [inline README.md]: Not a blocker for merge, but should we add language saying patches should be submitted to SSG and not this repository? If so, how should that language be worded?

File: meta/main.yml [hunk 3]
```diff
@@ -1,11 +1,7 @@
 galaxy_info:
-  author: your name
-  description: your description
-  company: your company (optional)
-
-  # If the issue tracker for your role is not on github, uncomment the
-  # next line and provide a value
-  # issue_tracker_url: http://example.com/issue/tracker
+  author: OpenSCAP
+  description: Open Source Security Compliance Solution
```
Comment (shawndwells) [inline meta/main.yml]: Since this playbook is specifically for the STIG, should that be reflected in the description?

File: tasks/main.yml [hunk 4]
```diff
@@ -1,2 +1,6690 @@
...
+###############################################################################
+
+
+# - hosts: localhost # set required host
+    - name: Check existence of yum on Fedora
+      stat:
+        path: /etc/yum.conf
+      register: yum_config_file
+      check_mode: no
+      when: ansible_distribution == "Fedora"
```
Comment (shawndwells) [inline tasks/main.yml]: If this playbook also supports Fedora, should main.yml be updated?

File: README.md (hunk 1 above)
Comment (WesleyCeraso) [reply README.md]: Well, the description will be automatically generated from SSG, so it's gonna be fixed by your PR.

File: meta/main.yml (hunk 3 above)
Comment (WesleyCeraso) [reply meta/main.yml]: Good point, I'll improve it

File: tasks/main.yml (hunk 4 above)
Comment (WesleyCeraso) [reply tasks/main.yml]: Fedora has its own ansible role, maybe the ansible remediation needs some improvement.

**Pull request closed without merging**